src/typings.d.ts

```diff
@@ -0,0 +1,2 @@
+// These are typings that do not have any type definitions and are therefore stubbed so that TS is happy
+declare module 'any-observable';
```

Comment:
Bleh. This is going away in the next version of cypher-query-builder. I was hoping to avoid dealing with it. It's needed because of our `Transaction` class right?

Reply:
Yep that's the only place it's being used